test/draft_test_primer_annotate.py

```diff
@@ -0,0 +1,60 @@
+from nose.tools import eq_, ok_
```

Comment:
You've got three versions of the same file. Delete the other ones, this is the right file because it's within the `test/` directory

Reply:
Also, nose is fine, the other option is to use `pytest`